ts/examples/vite-example/src/smelter-examples/dynamic/InputControls.tsx

```diff
@@ -0,0 +1,50 @@
+import type Smelter from '@swmansion/smelter-web-wasm';
+import type { InputsState } from './DynamicExample';
+import { MP4_AUDIO_ID, MP4_NO_AUDIO_ID } from './Controls';
+
+type InputProps = {
+  inputs: InputsState;
+  setInputs: (inputs: InputsState) => void;
+  smelter: Smelter;
+};
+
+export default function InputControls(props: InputProps) {
+  const toggleMp4WithAudio = async () => {
+    if (props.inputs.mp4WithAudio) {
+      await props.smelter.unregisterInput(MP4_AUDIO_ID);
+    } else {
+      await props.smelter.unregisterInput(MP4_AUDIO_ID).catch(() => {});
+      await props.smelter.registerInput(MP4_AUDIO_ID, {
+        type: 'mp4',
+        url: 'https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerEscapes.mp4',
+      });
+    }
+    props.setInputs({ ...props.inputs, mp4WithAudio: !props.inputs.mp4WithAudio });
+  };
+
+  const toggleMp4NoAudio = async () => {
+    if (props.inputs.mp4Silent) {
+      await props.smelter.unregisterInput(MP4_NO_AUDIO_ID);
+    } else {
+      await props.smelter.unregisterInput(MP4_NO_AUDIO_ID).catch(() => {});
+      await props.smelter.registerInput(MP4_NO_AUDIO_ID, {
+        type: 'mp4',
+        url: 'https://smelter.dev/videos/template-scene-race.mp4',
```

Comment:
This video is really short and ends abruptly which made me feel like something is wrong because the input suddenly disappeared. I would either use different video or display "Input finished" at the end